ts/IterableInboxMessageCell.tsx

```diff
@@ -166,6 +166,37 @@ const IterableInboxMessageCell = ({
 }: MessageCellProps) => {
    const position = useRef(new Animated.ValueXY()).current
 
+   let deleteSliderHeight = customizations.messageRow?.height ? customizations.messageRow.height: 120
+   
+   if(messageListItemLayout(last, rowViewModel)) {
+      deleteSliderHeight = messageListItemLayout(last, rowViewModel)[1]
```

Comment:
is this condition for last row? what is specific about it?

Reply:
If there is no specified custom row height, it defaults to a height of 120. The issue we were having is when the message row height is customized, the delete slider height was remaining the default of 120.

Reply:
This is for all message rows. I added the messageRow height as a value passed down to the IterableInboxMessageCell component (messageListItemLayout[0] is the custom layout, messageListItemLayout[1] is the custom message row height). This will be used to update the deleteSlider height in the IterableInboxMessageCell component. The dev app is updated to reflect this. If no custom message row height is provided, the delete slider will default to 120. Let me know if you need to sync for further explanation.